### DISCUSSION TITLE
Does the data updates along the time?

### DISCUSSION THREAD

**Discussion by ds-mauri** [meteostat/meteostat-python#172] [Q&A]
Guys, really dummy question.

If I installed the package, for instance, on june 1st, and I'm using it on 15th september. Does the weather data from this period is automatically updated or I have to always re-install the package to get the new predictions?

Thanks a lot!

**Comment by clampr** [comment]
Hi @ds-mauri, you always get the latest data. No re-install required 😉

**Reply by ds-mauri** [reply]
How do you guys do that? Just for curiosity! hahaha didn't know it was possible to keep a package updated without the need of the user to reinstall new versions (I don't have software eng. background, so don't know if this is usual).
